src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/DecisionOutcomeCallbackHandler.java

```diff
@@ -31,9 +35,13 @@ protected Map<String, Callback> callbacks() {
     }
 
     private CallbackResponse changeState(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData().toBuilder()
+            .build();
+        CaseData.CaseDataBuilder<?, ?> caseDataBuilder = caseData.toBuilder();
+        caseDataBuilder.businessProcess(BusinessProcess.ready(CREATE_DASHBOARD_NOTIFICATION_DECISION_OUTCOME));
```

Comment:
Please put this behind feature toggle, e.g:
`if(featureToggle.isCaseProgression()){ caseDataBuilder.businessProcess() }`